tests/data/transformer/14-mixed-operators/authorization-model.json

```diff
@@ -0,0 +1,118 @@
+{
+  "schema_version":"1.1",
+  "type_definitions": [
+    {
+      "type":"user",
+       "relations": {},
+       "metadata": null
+    },
+    {
+      "metadata": {
+        "relations": {
+          "allowed": {
+            "directly_related_user_types": [
+              {
+                "type":"user"
+              }
+            ]
+          },
+          "base": {
+            "directly_related_user_types": [
+              {
+                "type":"user"
+              }
+            ]
+          },
+          "blocked": {
+            "directly_related_user_types": [
+              {
+                "type":"user"
+              }
+            ]
+          },
+          "relation1": {},
+          "relation2": {},
+          "relation3": {}
```

Comment:
Should these be empty objects or `"directly_related_user_types": []`?

Reply:
Our current implementation is that it is an empty object.  However, I would say both interpretation can be considered correct.

Reply:
If both are correct, I wont touch the `dslToJSON` code here to remove `directly_related_user_types` from being added.